assets/src/api.ts

```diff
@@ -59,10 +66,21 @@ export const fetchRoutes = (): Promise<Route[]> =>
     })
 
 export const fetchShapeForRoute = (routeId: RouteId): Promise<Shape[]> =>
-  fetch(`/api/shapes/${routeId}`)
+  fetch(`/api/shapes/route/${routeId}`)
     .then(checkResponseStatus)
     .then(parseJson)
-    .then(({ data: shapes }: ShapesFroRouteResponse) => shapes)
+    .then(({ data: shapes }: ShapesForRouteResponse) => shapes)
+    .catch(error => {
+      // tslint:disable-next-line: no-console
+      console.error(error)
+      throw error
+    })
+
+export const fetchShapeForTrip = (tripId: TripId): Promise<Shape | null> =>
+  fetch(`/api/shapes/trip/${tripId}`)
+    .then(checkResponseStatus)
+    .then(parseJson)
+    .then(({ data: shape }: { data: Shape | null }) => shape)
     .catch(error => {
       // tslint:disable-next-line: no-console
       console.error(error)
```

Comment:
I'd rather put more thought into fixing this for all the api calls at once, rather than one by one for new calls. Followup task: [Properly handle errors in api calls.](https://app.asana.com/0/1152340551558956/1158195594242473)

Reply:
Fixing the other cases is a great followup task, but why not just do this one correctly right now?

Reply:
Because I don't want to add a special case for this function, and then immediately refactor it out.

Default values for all api calls where it makes sense are ready to go in #424 (built on top of this PR).

Reply:
I know we're having a discussion on #424 so it won't be ready to go immediately after this, but I don't think this PR needs to wait for it. The reason to return the default now in this PR would be so that it's in a better state for the long term if we don't follow up, but with that PR open already, I don't think that's a big risk.